README.md

```diff
@@ -8,22 +8,40 @@ This is a repository for the Jamaica mHealth CHT application project. To learn m
 
-Install cht-conf as guided [here](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/#2-install-cht-conf)
+- Install cht-conf as guided [here](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/#2-install-cht-conf)
+
+- **Optional**: Install Valid TLS Certificate.
+To install a valid certificate, open a terminal in the cht-core directory. Ensure the medic-os container is running and make this call: 
+```
+$./scripts/add-local-ip-certs-to-docker.sh
+```
 
-Optional: Install Valid TLS Certificate.
-To install a valid certificate, open a terminal in the cht-core directory. Ensure the medic-os container is running and make this call: `./scripts/add-local-ip-certs-to-docker.sh`
 More details on this [here](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/#5-optional-install-valid-tls-certificate)
 
 More information about CHT Local Environment Setup [here](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/)
 
 ## Loading the jamaica mHealth clinical app forms in CHT
 
-After following the steps above, you now have a running CHT instance on which to deploy the application.To deploy and load jamaica mHealth clinical app forms on the running CHT instance clone [mydhi-cht-config app](https://github.com/I-TECH-UW/mydhi-cht-config) run `cht --url=https://<username>:<password>@localhost --accept-self-signed-certs convert-app-forms upload-app-forms -- <app-form-name>` if a valid TLS certiifate was not used and `cht --url=https://<username>:<password>@<domain-name> convert-app-forms upload-app-forms -- <app-form-name>`
-if valid TLS certiifate was used eg `cht --url=https://<username>:<password>@192-168-5-70.my.local-ip.co convert-app-forms upload-app-forms -- <app-form-name>`.
+After following the steps above, you now have a running CHT instance on which to deploy the application.To deploy and load jamaica mHealth clinical app forms on the running CHT instance clone [mydhi-cht-config app](https://github.com/I-TECH-UW/mydhi-cht-config),navigate to the config folder and run 
+
+```
+$ cht --url=https://<username>:<password>@localhost --accept-self-signed-certs convert-app-forms upload-app-forms -- <app-form-name>
+```
+if a valid TLS certiifate was not used..
+
+Use 
+```
+$ cht --url=https://<username>:<password>@<domain-name> convert-app-forms upload-app-forms -- <app-form-name>
+```
+if valid TLS certiifate was used eg 
+```
+cht --url=https://<username>:<password>@192-168-5-70.my.local-ip.co convert-app-forms upload-app-forms -- <app-form-name>
```

Comment:
can you LIst the names of the forms `<app-form-name>`  to be used here in the read me??